src/robocoup/jobs.js

```diff
@@ -7,15 +8,21 @@ import {query} from '../lib/db';
 const scheduler = new Scheduler();
 export default scheduler;
 
-scheduler.add('00 00 09 * * 1', function() {
+scheduler.add('0 12 * * 1-5', function() {
   query('expertise_outstanding_for_all').then(users => {
+    const throttle = new Throttle({
+      requestsPerSecond: 1,
+      promiseImplementation: Promise
+    });
     const promises = users.map(({slack, outstanding}) => {
-      const plural = outstanding.length !== 1;
-      const message = heredoc.oneline.trim`
-        You have ${outstanding.length} outstanding expertise${plural ? 's' : ''} (${outstanding.join(', ')}).
-        Please update ${plural ? 'them' : 'it'} with the \`expertise update missing\` command.
-      `;
-      return sendDM(this, slack, message);
+      throttle.add(() => {
```

Comment:
Right now, the callback to `users.map` isn't returning anything. Does that mean that `promises` is just going to be an array of `undefined` values? If so, `Promise.all(promises)` won't wait for anything.

Does `throttle.add(fn)` return a promise? If so, that should be returned so that `Promise.all(promises)` works as-expected.

Otherwise, this code will need to be refactored!